### DISCUSSION TITLE
Custom Code - how to link images from assets

### DISCUSSION THREAD

**Discussion by alisonmwhite** [vanjarosoftware/Vanjaro.Platform#1590] [General]
hi,
i am testing some custom code design, and i need to know what URL is necessary to link images from the assets folder

i tried the full URL

https://vanjaros1.blob.core.windows.net/dcbzylfjvj/assets/images/webinar-tableau/

i also tried

/images/webinar-tableau/

thanks
Ali

**Comment by meetmandeep** [comment]
If you want to manually find the URL for an asset then you may do so from Menu > Site > Assets

Navigate to your file, use the item menu (3 dots) and click on Get URL
